src/components/implementation/no_interface/vkernel/vk_api.h

```diff
@@ -4,10 +4,10 @@
 #include "vk_types.h"
 
 /* extern functions */
-extern void vm_exit(void *);
 extern void vm_init(void *);
 extern void dom0_io_fn(void *);
 extern void vm_io_fn(void *);
+extern void *__inv_vkernel_serverfn(int a, int b, int c);
```

Comment:
I'm sorry, this is not really a generic library, it is very specific to vkernel component and the main focus for `vk_api.c/h` is modularizing the code.